exercises/perfect-numbers/metadata.yml

```diff
@@ -0,0 +1,5 @@
+  
```

Comment:
This file shouldn't be added to this repo. It's only used to generate the README but it'll be taken from the problem-spec repo when doing that.